### PR TITLE
Adding unit tests Alignment-INDOT and SectionedSurface-INDOT

### LINKED ISSUE #73: Pull request #71 IFC error
See #71 : I get the following error from KIT checker for the alignment file:

`#56=IFCSHAPEREPRESENTATION(#55,'Axis','Curve2D',(#50))`

> where rule 'IFCSHAPEREPRESENTATION.CORRECTITEMSFORTYPE' evaluates to FALSE

`#107=IFCSHAPEREPRESENTATION(#55,'Axis','Curve3D',(#106))`

> where rule 'IFCSHAPEREPRESENTATION.CORRECTITEMSFORTYPE' evaluates to FALSE

*EDIT: markdown formatting*

### PR DESCRIPTION
**Pull request by larswik**
## Unit test

Continuing from #66 

## Covered topics

*tick one/many*

- [x] project setup
- [ ] georeferencing
- [x] alignment semantics
- [x] alignment geometry
- [ ] linear placement
- [ ] project breakdown structure / spatial structure
- [ ] terrain
- [ ] triangulated irregular network
- [ ] open cross profile
- [ ] sectioned surface & solid
- [ ] pavement & course
- [ ] surface feature
- [ ] earthworks
- [ ] geotechnics
- [ ] longitudinal products (barriers, guardrail, ...)
- [ ] interchange
- [ ] structural
- [ ] drainage
- [ ] signage
- [ ] road furniture
- [ ] rail furniture
- [ ] ports & waterways furniture
- [ ] bridge furniture

### CODE REVIEW

File: Alignment-INDOT/PR_Twin_Branch_section_alignment.ifc [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+#15= IFCUNITASSIGNMENT((#12,#14));
+#16= IFCPROJECT('3AmF9rizLFdPhI8nWCsL67',$,'UNKNOWN PROJECT',$,$,$,$,(#51),#15);
+#17= IFCCARTESIANPOINT((0.0,0.0,0.0));
+#18= IFCAXIS2PLACEMENT3D(#17,#4,#2);
+#19= IFCLOCALPLACEMENT($,#18);
+#20= IFCSITE('2PlVuc8pjDywr$PZgmi13a',$,$,$,$,#19,$,$,.COMPLEX.,$,$,$,$,$);
+#21= IFCLOCALPLACEMENT($,#5);
+#22= IFCALIGNMENT('0GLEj6tMb19u3SzxDDGgUo',$,'PR_Twin_Branch_section',$,$,#21,$,$,$);
+#23= IFCRELCONTAINEDINSPATIALSTRUCTURE('0Y93m4$W95IhXby4BL_2BW',$,'Site','Site Container for Elements',(#22),#20);
+#24= IFCALIGNMENTHORIZONTAL('2mv6_WjB5EnBUjc2pdivhk',$,$,$,$,#21,#57,0.0,(#));
```
Comment (pjanck) [inline Alignment-INDOT/PR_Twin_Branch_section_alignment.ifc]: See last parameter - there seems to be an error with the IFC writing procedure.

File: Alignment-INDOT/readme.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    - `IfcLine`
+    - `IfcCircle`
+    - `IfcGradientCurve`
+    - `IfcPolynomialCurve`
+
+### Situation
+
+The figure below illustrates the horizontal and vertical alignments.
+
+![](./Illustration.PNG)
```
Comment (pjanck) [inline Alignment-INDOT/readme.md]: For the sake of automatic report production, please always use "one folder up, one folder down" for the figures:

```suggestion
![](../Alignment-INDOT/Illustration.PNG)
```

File: Alignment-INDOT/readme.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+2. The resulting IFC file with an alignment and its geometric representation.
+
+### Supporting files
+
+Following files correspond to this scenario:
+
+| Filename                               | Description                           |
+| -------------------------------------- | ------------------------------------- |
+| `PR_Twin_Branch_section_alignment.xml` | LandXML file with the alignment       |
+| `PR_Twin_Branch_section_alignment.ifc` | Sample IFC 4.3 file for the alignment |
```
Comment (pjanck) [inline Alignment-INDOT/readme.md]: I'd expect to have the figures listed here as well.

**Comment by larswik** [comment]
Ok to merge? @pjanck

**Pull request merged**